lib/graphql/resolver.ex

```diff
@@ -2536,6 +2543,33 @@ defmodule AshGraphql.Graphql.Resolver do
     child_complexity + 1
   end
 
+  def resolve_node(%{arguments: %{id: id}} = resolution, type_to_api_and_resource_map) do
+    with {:ok, {type, primary_key}} <- AshGraphql.Resource.decode_relay_id(id),
+         {:ok, {api, resource}} <- Map.fetch(type_to_api_and_resource_map, type) do
+      # TODO: what if there's no get query? It should probably not implement the node interface
+      query = get_query(resource)
+
+      # We pass relay_ids? as false since we pass the already decoded primary key
+      put_in(resolution.arguments.id, primary_key)
+      |> resolve({api, resource, query, false})
+    else
+      _ ->
+        # TODO: is this return value ok?
+        Absinthe.Resolution.put_result(resolution, {:error, "Invalid primary key"})
```

Comment:
Not sure what the return value should be, probably an `Ash.Error.Framework.AssumptionFailed`. Although we could avoid the case entirely by requiring that each resource has a `get` query at compile time.